parl/env/compat_wrappers.py

```diff
@@ -36,6 +38,18 @@ def get_gym_version(version_str=gym.__version__):
 
 
 class CompatWrapper(gym.Wrapper):
+    """ Compatible for different versions of gym, especially for `step()` and `reset()` APIs.
+
+        .. code-block:: python
+        # old version of gym APIs
+        observation = env.reset()
+        observation, reward, terminated, truncated, info = env.step(action)
+        # new version of gym APIs
+        observation, info = env.reset()
+        observation, reward, done, info = env.step(action)
```

Comment:
Looks like you have mixed up old and new gym APIs. Maybe you can explain at the end of the doc that the new version of the gym env can be used in the same way as the old version of gym, after being wrapped by ComptWrapper

Reply:
done